lib/cli/src/Cardano/CLI.hs

```diff
@@ -1577,22 +1577,15 @@ data Verbosity
     deriving (Eq, Show)
 
 data LogOutput
-    = LogToStdout Severity
+    = LogToStdouterr Severity
+    -- ^ Log [sev .. Critical] to either stderr or stdout.
+    -- To stderr for (>= Notice), and to stdout for (< Notice).
```

Comment:
```suggestion
    = LogToStdStreams Severity Severity
    -- ^ Log to console, with the given minimum 'Severity' for @stdout@ messages
    -- and minimum 'Severity' for @stderr@ messages.
```

Here's a better name, and we may as well make parameters for both severities - less explaining to do in comments.

Reply:
Name — 👍 , but I don't think we should take parameters for both parameters. Then we'd need to add complexity to the call-sites (local-cluster, cardano-wallet, integration)